data/json/items/melee.json

```diff
@@ -2116,9 +2116,9 @@
     "looks_like": "mace",
     "material": [ "aluminum", "wood" ],
     "techniques": [ "SWEEP" ],
-    "flags": [ "NONCONDUCTIVE" ],
+    "flags": [ "NONCONDUCTIVE", "FRAGILE_MELEE" ],
```

Comment:
Like the axe, it seems like its be pretty easy to make one of these robust enough to not deserve the fragile flag.